samples/charts/ScrollToTop.js

```diff
@@ -0,0 +1,31 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+import React, { Component } from 'react';
+import {withRouter} from 'react-router-dom';
+
+class ScrollToTop extends Component {
```

Comment:
Please explain the purpose of this React component.
To me this look like a behavior of a component(s), father a component it self.

Reply:
https://github.com/ReactTraining/react-router/blob/v4.2.1/packages/react-router-dom/docs/guides/scroll-restoration.md

Reply:
ok